src/app/services/royal-shaving/page.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { RoyalShaving } from '@/Features/RoyalShaving';
+import { Footer } from '@/components/Footer';
+
+const RoyalShavingPage = async () => {
+  const [headerListResponse, barbershopsInfoResponse, footerListsResponse] = await Promise.all([
+    fetch('http://localhost:3000/api/headerList'),
+    fetch('http://localhost:3000/api/barbershopsInfo'),
+    fetch('http://localhost:3000/api/footerLists'),
+  ]);
+  const [
+    { headerMainList },
+    { branchesTotal, countriesTotal, capitalsTotal },
+    { footerMainList, footerServicesList, footerInfoList },
+  ] = await Promise.all([
+    headerListResponse.json(),
+    barbershopsInfoResponse.json(),
+    footerListsResponse.json(),
+  ]);
+
+  return (
+    <main className="overflow-hidden flex md:container md:max-w-screen-2xl mx-auto flex-col">
+      <RoyalShaving mainNavList={headerMainList} />
+      <Footer
```

Comment:
I made the same semantic mistake. `<main>` tag shouldn't contain `<footer>` one.